config/packages/security.yaml

```diff
@@ -1,8 +1,9 @@
 security:
     encoders:
         # Our user class and the algorithm we'll use to encode passwords
+        # 'auto' means to let Symfony choose the best possible password hasher (Argon2 or Bcrypt)
```

Comment:
`argon2i` and `bcrypt` are deprecated since 4.3
The new 'auto' choice list is `sodium` or `native`

Reply:
ref: https://github.com/symfony/symfony/blob/c5772ae9bc3d9a238a9d03b97afc773ce16b93ee/src/Symfony/Component/Security/Core/Encoder/EncoderFactory.php#L87-L89

Reply:
which are themselves using bcrypt or argon2i as algorithm.

Reply:
Yes, I meant the encoder names used for this `auto` option.